app/src/main/java/HomeMultiMode.kt

```diff
@@ -0,0 +1,4 @@
+package com.teamfillin.fillin
+
+const val multi_type1 = 1
+const val multi_type2 = 2
```

Comment:
코틀린 코딩 컨벤션에서 상수는 UPPER_SNAKE_CASE로 적습니다

```suggestion
const val MULTI_TYPE_1 = 1
const val MULTI_TYPE_2 = 2
```

그리고 MULTI_TYPE_1은 어떤 의미를 가지고 있지 않은 것 같습니다. 어떤 용도로 만드신거죠?